README.md

```diff
@@ -1 +1,85 @@
-# leagacyProject-IFL-Science
\ No newline at end of file
+# leagacyProject-IFL-Science
+
+## Main Idea:
```

Comment:
Could you please add more details for the main idea ??

Reply:
Thank you Mays I will add all the main aspects related to what we did in this project, if you  want me to add any other details please tell me to do the last version